WebServices/AzureSignalR/ChatClient/Model/NegotiateInfo.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+
+namespace ChatClient.Model
+{
+    public class NegotiateInfo
+    {
+        public Uri Url
+        {
+            get;
+            set;
+        }
+
+        public string UrlWithoutQuery
+        {
+            get
+            {
+                if (Url == null)
+                {
+                    return string.Empty;
+                }
+
+                string pathWithoutQuery = Url.PathAndQuery.Substring(
+                    0,
+                    Url.PathAndQuery.IndexOf("?"));
```

Comment:
VSMac squiggles this line, because `IndexOf` is culture aware, and so should have a `StringComparison` argument. While I'm not too bothered about this, just wondering if there's a setting you can turn on in VS so you see such squiggles/

Reply:
I did a couple searches and poked through editor settings, I couldn't find anything about generating warnings/suggestions about this for VS PC.